Samples/HelloWorld.NetCore/src/HelloWorld.Grains/HelloGrain.cs

```diff
@@ -8,8 +9,16 @@ namespace HelloWorld.Grains
     /// </summary>
     public class HelloGrain : Orleans.Grain, IHello
     {
+        private readonly ILogger logger;
+
+        public HelloGrain(ILogger<HelloGrain> logger)
+        {
+            this.logger = logger;
+        }  
+
         Task<string> IHello.SayHello(string greeting)
         {
+            logger.LogWarning($"SayHello message received: greeting = '{greeting}'");
```

Comment:
I know it's nitpicking, but for a new-comer, receiving a warning might be surprising, they might think something went wrong. Could you change to informational?

Reply:
Done, I started with Info level but changed to warning to standout between all the messages, but it definitely should be and information level message. Thanks.